apps/omg_watcher/lib/omg_watcher/api/in_flight_exit.ex

```diff
@@ -69,6 +72,7 @@ defmodule OMG.Watcher.API.InFlightExit do
 
   This delegates directly to `OMG.Watcher.ExitProcessor` see there for details
   """
+  @decorate measure_event()
```

Comment:
Wouldn't it make more sense to measure the event from `OMG.Watcher.Web.Controller.InFlightExit.prove_canonical/2`?

Reply:
yes, it's more sense.
and this is already happening, then it is opened appsignal transaction with namespace web

Reply:
so we're measuring the same thing twice, just the namespace is different?

Reply:
o start the measurements you have to call appsignal transaction, and the event can check how the transaction time will spread